regional_mom6/regional_mom6.py

```diff
@@ -1624,6 +2249,20 @@ def setup_run_directory(
             ## In case there is additional forcing (e.g., tides) then we need to modify the run dir to include the additional forcing.
             overwrite_run_dir = False
 
+        # Check if we can implement tides
+        if with_tides:
+            tidal_files_exist = any(
+                "tidal" in filename
+                for filename in (
+                    os.listdir(Path(self.mom_input_dir / "forcing"))
+                    + os.listdir(Path(self.mom_input_dir))
+                )
+            )
```

Comment:
```suggestion
            tidal_files_exist = any(Path(self.mom_input_dir).rglob("*tidal*"))
```

Probably something like this instead? Not sure if you expect any other subdirs